src/Microsoft.ManagedZLib/benchmarks/Microsoft.ManagedZLib.Benchmarks.csproj

```
@@ -7,9 +7,14 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="C:\Users\t-vivianadue\.nuget\packages\system.io.compression.testdata\8.0.0-beta.23326.1\contentFiles\any\any\DeflateTestData\~%24stDocument.doc" />
```

This is hardcoded to your machine. Can you try setting this to a folder that would work for anyone?:

I think this should work:
```suggestion
    <Compile Remove="$(NuGetPackageRoot)\system.io.compression.testdata\8.0.0-beta.23326.1\contentFiles\any\any\DeflateTestData\~%24stDocument.doc" />
```

---

It works.